zstacklib/zstacklib/utils/sanlock.py

```
@@ -11,10 +21,50 @@
+    @linux.retry_if_unexpected_value([], times=6, sleep_time=3)
+    def retry_if_corrupted(func, *args, **kwargs):
+        pass
+
+
+    def wrap(f):
+        @functools.wraps(f)
+        def inner(*args, **kwargs):
+            try:
+                return f(*args, **kwargs)
+            except Exception as e:
+                if retry(args, str(e)):
```

Ruff (0.8.2)

Undefined name `retry`

(F821)